item/historical/scripts/S012.ipynb

```diff
@@ -0,0 +1,3078 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import json\n",
+    "from pprint import pprint\n",
+    "from item.historical.scripts.util.managers.dataframe import DataframeManager\n",
+    "from item.historical.scripts.util.managers.dataframe import ColumnName\n",
+    "from item.historical.scripts.util.managers.country_code import CountryCodeManager\n",
+    "from item.common import paths"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Variables used across the noteboook and not changed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATASET_ID = \"S012\"\n",
```

Comment:
Why is this script called 'S...' instead of 'T...'? The [documentation](https://transportenergy.readthedocs.io/en/latest/historical.html) should provide enough information for a newcomer to understand this nomenclature and what the scripts contain, _without_ having to open the individual notebooks.